Update plugins.txt

Sledgehamma opened the pull request:
1. Übersetzungen angefangen

Commit:
Update plugins.txt

1. Übersetzungen angefangen

vakilando commented on plugins.txt:
```diff
@@ -1,28 +1,28 @@
+Next=Nächster
+No release notes available=Keine Veröffentlichungshinweise vorhanden
+Plugin File=Plugin Datei
+Plugin Update Check=Plugin 
```
eher "Prüfe auf Plugin-Aktualisierung"?

ich777 replied:
Plugins Updaten

vakilando commented on plugins.txt:
```diff
@@ -1,28 +1,28 @@
+Support Thread=Unterstützungs Thema
+Uninstall=Deinstallieren
+Update All Plugins=Aktualisiere alle Plugins
+Upgrade=auf eine höhere Stufe bringen
```
oder einfach nur Aktualisieren? 
Ich weiß nicht ob in diesem Kontext der Unterschied zwischen update/upgrade relevant ist, aber wenn ja fällt mir auch grad nix passenderes ein...

ich777 replied:
Aktualisieren wäre gut.

Sledgehamma replied:
Ich kann leider nicht im GUI sehen, ob da ein echter Unterschied zwischen Update und Upgrade ist. Falls nicht, wäre ich auch für "Aktualisieren".

ich777 replied:
Muss nicht alles perfekt am anfang sein, plausibilität prüfe ich dann nächste woche.

Commit:
Update plugins.txt

"Plugin" durch "Prüfe auf Plugin-Aktualisierung" bei "Plugin Update Check" ersetzt
